### PR TITLE
style(hero): Transform svg in css

### LINKED ISSUE #256: Bug: Broken design on very large screens
### Describe the bug

On large screens, the nav bar keeps expanding out of their positions. There should be a max width set for the nav bar as we have the same restriction on other contents on the page. Also, the hero's background color should also extend and cover its space just like the hot repositories section

### Steps to reproduce

1. Go to [https://hot.opensauced.pizza/](https://hot.opensauced.pizza/) on a large screen or zoom out on your laptop to 50%
large screen screenshot

### Affected services

hot.opensauced.pizza

### Platforms

Desktop

### Browsers

Chrome, Firefox, Edge

### Environment

Production, Development

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by NsdHSO**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

closes #256

## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by hokagedemehin** [comment]
IMHO, I think the nav should have a certain max width and not always expand out based on the screen size.

The same way the rest of the page has a max width it can expand to

**Comment by NsdHSO** [comment]
Ok, I understand what do you mean, But just a second to see what say @pixelsbyeryc

File: src/components/styles/grandBackground.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
...xsC8BuYyIzBmYk8L8YzT4CFA+QjRiF43qvjnGsTBkDMkfmYvycgYsX5dCkPlk0sWsbLF9MiEY+cxAc/yFixfLPHQqOg0Z0vuY8Z0QXU34fkkpfgMJWcHbGeaGFi9E432yQIumw5mLwlY7XLGgQoMXSxTqvirdIrshbxjbfKxrHSxGHS8YxnnJ9BmRCK6iY5EnIkxaXjfBGqROMJBBi6SKVg9Fi/BIo6YBMYsXkxJe+NTZwlBmNkoVysGLdK3otaSvJjK8pyvK0nKtMpJD4CYgQguhjEmkreT99IxdNwHPyaDsjkXU5CK4p6I7DDl/0xBpYg8HPmrWY/SGThmBiOgoFCVvZeiUjGtUkJFLXRcpP5br03w0KBERiCH6LpNjj99sitMkkpJCkbqtJR+K8LWsq0SQvpIiMX2RtpIIUCPocP+JAt96o4yb04SUnZCUJ6YEWC/wDonO870TyQYkgYbnYg0pP5bI5K1zXwQQ+pHM34i8L6ZWtEYMYYullvynYQI52Iw5i9c6b0PjWmlgQXPH0O5i/Jha6I03qvfQoVpoUhmQrlYXKX2SsGlb65CJxAP/eFge4hSUlaZsORchMchisGZM4cXw2jbY3SDSkreREPok547S9J66zYMH4mH2CDnfHfXS0lJ/m8gRFjeD7rFi0ZJB8xCSSd0D/xI5Bmdj7ge+++Ub8uAiFz5FJS1hWJsQjMsztwZEZBwSv3EsoyVlWdI+bL/oJLFj9ELlLV98idkYvrrCQZMsECybBiB/kfNe2FgMR51jSSjAjIhBA4gMQ+j6Cw+oQYqBG6XgfDeuM2yrTBBjBkhNeDEcL9Um6IS+CMq1ujWCFIQo0MfQTbD5YznOkoVqYEhhv2yF+hAMGykD8JsHPVWUaHzNsbE915mfMZYTgaHg3V921vgFo+43y3og91ZkjY/ci772EKFf8AcKwpazpPusk+9E43onClrZG6Fv2yUtV9zlk5lylwn4MpXH97G8g0PzZ5z9ivfbib1IyoX4v3uP8AiD95yxlTwla8G+R8p/h5/hW7K82NMaa9SdTF7M6G9gtUCMH90sSNG/gCB63DEf8A6NMV/CO38MfgF4c+42qf4ef4ecJ92MJScJ9ad8CRPvNpn1n0OGMg61hXrNqYP6pLORltvfHi2INaS8aSUjCMIz//2Q=='/%3E%3C/defs%3E%3C/svg%3E");
```
Comment (0-vortex) [inline src/components/styles/grandBackground.css:1]: This is just doing `base64` on the actual SVG image, it doesn't <count> as rewriting it to CSS

File: src/components/GradBackground.tsx [hunk 2]
```diff
@@ -1,14 +1,16 @@
 import { ReactNode } from 'react'
+import './styles/grandBackground.css'
```
Comment (0-vortex) [inline src/components/GradBackground.tsx:2]: This is tailwind, the correct way to do this is by re-creating the gradient based on the docs available at: https://tailwindcss.com/docs/gradient-color-stops

**Review by 0-vortex — CHANGES_REQUESTED** [review]

**Comment by 0-vortex** [comment]
The target branch for this should be `main`, make sure you keep your fork synced with upstream!

**Comment by bdougie** [comment]
@open-sauced/triage check it out

**Comment by bdougie** [comment]
> IMHO, I think the nav should have a certain max width and not always expand out based on the screen size.

I agree but this would provide a level of scope creep that should be pushed to a new issue and addressed in a future PR.

**Comment by pixelsbyeryc** [comment]
> IMHO, I think the nav should have a certain max width and not always expand out based on the screen size.
> 
> The same way the rest of the page has a max width it can expand to

@NsdHSO I agree with @hokagedemehin... Having a `max-width` will contain the contents, and help it look good on bigger screens. Our eyes scan from side to side, so imagine on a ultra-wide 49" monitor how it'd be to scan across.

But as @bdougie, we can move on and open another issue specifically for this. 🙌 

**Suggested `max-width`:** 1024px, 1280px, or 1440px... It could even be a combination of these, that would increase as the screen size gets bigger, eg:
- Viewport width > 1200px 👉  `max-width: 1024px`
- Viewport width > 1400px 👉  `max-width: 1280px`
- Viewport width > 1600px 👉  `max-width: 1440px`

**Review by bdougie — APPROVED** [review]
Reviewed live on air and saw open conversations resolved. LGTM

**Pull request merged**